apps/closer/components/ErrorBoundary/ErrorBoundary.tsx

```
@@ -0,0 +1,28 @@
+
+  static getDerivedStateFromError(error: Error) {
+    console.error(error);
+    return { hasError: true, error };
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return (
+        <>
+          <Heading>Something went wrong.</Heading>;
+        </>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)